support/scheduler-client/client_test.go

```diff
@@ -0,0 +1,425 @@
+//
+// Copyright (c) 2017 Tencent
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package scheduler
+
+import (
+	"encoding/json"
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+	"testing"
+)
+
+// Test common const
+const (
+	TestUnexpectedMsg                    = "unexpected result"
+	TestUnexpectedMsgFormatStr           = "unexpected result, active: '%s' but expected: '%s'"
+	TestUnexpectedMsgFormatStrForBoolVal = "unexpected result, active: '%t' but expected: '%t'"
+)
+
+// Test comm api path
+const (
+	TestExpectedScheduleApiPath      = "/api/v1/schedule"
```

Comment:
The same value as ScheduleApiPath and ScheduleEventApiPath, remove TestExpectedScheduleApiPath and TestExpectedScheduleEventApiPath